tests/upload_test.go

```diff
@@ -99,21 +141,24 @@ var _ = Describe("[rfe_id:138][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			Expect(err).ToNot(HaveOccurred())
 			Expect(found).To(BeTrue())
 
-			By("Verify content")
-			same, err := f.VerifyTargetPVCContentMD5(f.Namespace, pvc, utils.DefaultImagePath, utils.UploadFileMD5, false)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(same).To(BeTrue())
-			fileSize, err := f.RunCommandAndCaptureOutput(pvc, "stat -c \"%s\" /pvc/disk.img")
-			Expect(err).ToNot(HaveOccurred())
-			Expect(fileSize).To(Equal("1073741824")) // 1G
+			if !isBlockVolume {
```

Comment:
No way for now to check content on block?

Reply:
still working on this...

Reply:
done